docs/content.zh/docs/connectors/datastream/dynamodb.md

```diff
@@ -0,0 +1,170 @@
+
+val elementConverter = new CustomElementConverter();
+
+val dynamoDbSink =
+   DynamoDbSink.<InputType>builder()
+           .setDynamoDbProperties(sinkProperties)              // Required
+           .setTableName("my-dynamodb-table")                  // Required
+           .setElementConverter(elementConverter)              // Required
+           .setOverwriteByPartitionKeys(singletonList("key"))  // Optional    
+           .setFailOnError(false)                              // Optional
+           .setMaxBatchSize(25)                                // Optional
+           .setMaxInFlightRequests(50)                         // Optional
+           .setMaxBufferedRequests(10_000)                     // Optional
+           .setMaxTimeInBufferMS(5000)                         // Optional
+           .build()
+
+flinkStream.sinkTo(dynamoDbSink)
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Configurations
+
+Flink's DynamoDB sink is created by using the static builder `DynamoDBSink.<InputType>builder()`.
+
+1. __setDynamoDbProperties(Properties sinkProperties)__
+   * Required.
+   * Supplies credentials, region and other parameters to the DynamoDB client.
+2. __setTableName(String tableName)__
+   * Required.
+   * Name of the table to sink to.
+3. __setElementConverter(ElementConverter<InputType, DynamoDbWriteRequest> elementConverter)__
+   * Required.
+   * Converts generic records of type `InputType` to `DynamoDbWriteRequest`.
+4. __setOverwriteByPartitionKeys(List<String> partitionKeys)__
+   * Optional.
+   * Used to deduplicate write requests within each batch pushed to DynamoDB.
+5. _setFailOnError(boolean failOnError)_
+   * Optional. Default: `false`.
+   * Whether failed requests to write records are treated as fatal exceptions in the sink.
+6. _setMaxBatchSize(int maxBatchSize)_
+   * Optional. Default: `500`.
+   * Maximum size of a batch to write.
+7. _setMaxInFlightRequests(int maxInFlightRequests)_
+   * Optional. Default: `50`.
+   * The maximum number of in flight requests allowed before the sink applies backpressure.
+8. _setMaxBufferedRequests(int maxBufferedRequests)_
+   * Optional. Default: `10_000`.
+   * The maximum number of records that may be buffered in the sink before backpressure is applied.
+9. _setMaxBatchSizeInBytes(int maxBatchSizeInBytes)_
+   * N/A.
+   * This configuration is not supported, see [FLINK-29854](https://issues.apache.org/jira/browse/FLINK-29854).
+10. _setMaxTimeInBufferMS(int maxTimeInBufferMS)_
+* Optional. Default: `5000`.
+* The maximum time a record may stay in the sink before being flushed.
+11. _setMaxRecordSizeInBytes(int maxRecordSizeInBytes)_
+* N/A.
+* This configuration is not supported, see [FLINK-29854](https://issues.apache.org/jira/browse/FLINK-29854).
+12. _build()_
+* Constructs and returns the DynamoDB sink.
+
+## Element Converter
+
+An element converter is used to convert from an object in the DataStream to a type that can be written to the
```

Comment:
`object` sounds a little out of place here. Maybe `record`?

Also, "to a type that can be written to the destination by the sink" is a little generic. Should we say instead "to a DynamoDbWriteRequest which the sink will write to the destination DynamoDB table"?